packages/atlas/src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -326,6 +328,14 @@ export const MyVideosView = () => {
               </Button>
             )}
           </TabsContainer>
+          {currentChannel && (
+            <StyledBanner
+              dismissibleId="yppSyncInfo"
+              title="YouTube Sync is enabled"
+              icon={<SvgAlertsInformative24 />}
+              description={`Whenever you upload video to ${currentChannel.title} YouTube channel, it will automatically appear here after a ashort while. You can change this setting in your YouTube Partner Program dashboard.`}
```

Comment:
```suggestion
              description={`Whenever you upload video to ${currentChannel.title} YouTube channel, it will automatically appear here after a short while. You can change this setting in your YouTube Partner Program dashboard.`}
```